fragments/neues/entry.php

```diff
@@ -49,7 +49,7 @@
 
 
                 <!-- Post Content -->
-                <?php if ($post->getDescription()) : ?>
+                <?php if ('' < $post->getDescription()) : ?>
```

Comment:
Das ist so gültig? Nicht eher so:
```suggestion
                <?php if ('' !== $post->getDescription()) : ?>
```

Reply:
In dem Fall ist das wohl egal, da getDescription string zurückliefert. Wenn die Funktion sowas wie `false|string` liefert oder `null|string` sollte man === bzw. !=== benutzen, insb. wenn man RexStan auf "strict" peüfen lässt.